packages/react-native-media-player/android/src/main/java/com/yajananrao/mediaplayer/MediaPlayerService.java

```diff
@@ -0,0 +1,590 @@
+                        | PlaybackStateCompat.ACTION_STOP | PlaybackStateCompat.ACTION_SKIP_TO_NEXT
+                        | PlaybackStateCompat.ACTION_SKIP_TO_PREVIOUS);
+            } else if (state == PlaybackStateCompat.STATE_PAUSED) {
+                playbackstateBuilder.setActions(PlaybackStateCompat.ACTION_PLAY_PAUSE | PlaybackStateCompat.ACTION_PLAY
+                        | PlaybackStateCompat.ACTION_STOP | PlaybackStateCompat.ACTION_SKIP_TO_NEXT
+                        | PlaybackStateCompat.ACTION_SKIP_TO_PREVIOUS);
+            } else {
+                playbackstateBuilder.setActions(PlaybackStateCompat.ACTION_PLAY_PAUSE | PlaybackStateCompat.ACTION_PAUSE
+                        | PlaybackStateCompat.ACTION_STOP | PlaybackStateCompat.ACTION_SKIP_TO_NEXT
+                        | PlaybackStateCompat.ACTION_SKIP_TO_PREVIOUS);
+            }
+            if (mMediaPlayer != null) {
+                position = mMediaPlayer.getCurrentPosition();
+                Log.d(TAG, "setMediaPlaybackState: position " + position);
+
+            }
+            playbackstateBuilder.setState(state, position, 1.0f);
+            mMediaSessionCompat.setPlaybackState(playbackstateBuilder.build());
+        } catch (Exception e) {
+            Log.e(TAG, "setMediaPlaybackState: " + e.toString());
+        }
+
+    }
+
+
+    private boolean successfullyRetrievedAudioFocus() {
+        int result;
+        AudioManager audioManager = (AudioManager) getSystemService(Context.AUDIO_SERVICE);
+        if (Build.VERSION.SDK_INT >= 26) {
+            focus = new AudioFocusRequest.Builder(AudioManager.AUDIOFOCUS_GAIN).setOnAudioFocusChangeListener(this)
+                    .setAudioAttributes(new AudioAttributes.Builder().setUsage(AudioAttributes.USAGE_MEDIA)
+                            .setContentType(AudioAttributes.CONTENT_TYPE_MUSIC).build())
+                    .setWillPauseWhenDucked(false).build();
+
+            result = audioManager.requestAudioFocus(focus);
+        } else {
+            // noinspection deprecation
+            result = audioManager.requestAudioFocus(this, AudioManager.STREAM_MUSIC, AudioManager.AUDIOFOCUS_GAIN);
+        }
+
+        return result == AudioManager.AUDIOFOCUS_GAIN;
+    }
+
+    // Not important for general audio service, required for class
+    @Nullable
+    @Override
+    public BrowserRoot onGetRoot(@NonNull String clientPackageName, int clientUid, @Nullable Bundle rootHints) {
+        if (TextUtils.equals(clientPackageName, getPackageName())) {
+            return new BrowserRoot("RNAudio", null);
+        }
+        return null;
+    }
+
+    // Not important for general audio service, required for class
+    @Override
+    public void onLoadChildren(@NonNull String parentId, @NonNull Result<List<MediaBrowserCompat.MediaItem>> result) {
+        result.sendResult(null);
+    }
+
+    @Override
+    public void onAudioFocusChange(int focusChange) {
+        switch (focusChange) {
```

Comment:
Codacy found an issue: [Switch statements should have a default label](https://app.codacy.com/manual/yajananrao/Nadha/pullRequest?prid=5728824)